src/Symfony/Component/Form/NativeRequestHandler.php

```diff
@@ -27,11 +28,23 @@ class NativeRequestHandler implements RequestHandlerInterface
     private $serverParams;
 
     /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var null|string
+     */
+    private $translationDomain;
+
+    /**
      * {@inheritdoc}
      */
-    public function __construct(ServerParams $params = null)
+    public function __construct(ServerParams $params = null, TranslatorInterface $translator = null, $translationDomain = null)
     {
         $this->serverParams = $params ?: new ServerParams();
+        $this->translator = $translator;
```

Comment:
You should create a translator if `null` is passed imo, since this can be used with Form as standalone component.